keyboards/kprepublic/cstc40/daughterboard/daughterboard.c

```diff
@@ -0,0 +1,71 @@
+/* Copyright 2022 bdtc123
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include "quantum.h"
+
+#ifdef RGB_MATRIX_ENABLE
+const is31_led g_is31_leds[RGB_MATRIX_LED_COUNT] = {
+    { 0, L_1,  K_1,  J_1 },
+    { 0, L_2,  K_2,  J_2 },
+    { 0, L_3,  K_3,  J_3 },
+    { 0, L_4,  K_4,  J_4 },
+    { 0, L_5,  K_5,  J_5 },
+    { 0, L_6,  K_6,  J_6 },
+	{ 0, L_7,  K_7,  J_7 },
```

Comment:
```suggestion
    { 0, L_7,  K_7,  J_7 },
```

Reply:
Thank you for this suggestion. I reviewed all the files to check and correct identation.